pdf/pdfcanvas.cpp

```diff
@@ -78,6 +89,12 @@ class PDFCanvas::Private
     float spacing;
 
     QRectF visibleArea;
+    bool rememberPagePosition;
+    QString dSource;
```

Comment:
Better not abbreviate names unless really necessary. Then again is the whole member variable needed when it can be fetched with document pointer?